editor/shader/shader_create_dialog.cpp

```diff
@@ -432,31 +441,33 @@ void ShaderCreateDialog::config(const String &p_base_path, bool p_built_in_enabl
 }
 
 String ShaderCreateDialog::_validate_path(const String &p_path) {
-	String p = p_path.strip_edges();
+	ERR_FAIL_COND_V(current_type >= type_data.size(), TTR("Invalid shader type selected."));
```

Comment:
```suggestion
	ERR_FAIL_COND_V(current_type >= type_data.size(), "Invalid shader type selected.");
```
Don't translate errors.